Link to Profile section at first use (#204)
ietf-rats-wg/draft-ietf-rats-corim: draft-ietf-rats-corim.md

* A locator, which allows discovery of possibly related RIMs

> * A profile identifier, which is used to interpret the information contained in the enclosed tags. A profile allows the base CoRIM schema to be customised to fit a specific Attester, though it MUST NOT alter the base CoRIM schema's syntax and semantics. For more, see {{sec-corim-profile-types}},

Reviewer: ```suggestion
* A profile identifier, which is used to interpret the information contained in the enclosed tags.
A profile allows the base CoRIM schema to be customised to fit a specific Attester, though it MUST NOT alter the base CoRIM schema's syntax and semantics.
For more, see {{sec-corim-profile-types}},
```

* A locator, which allows discovery of possibly related RIMs

* A profile identifier, which is used to interpret the information contained in the enclosed tags.

> A profile allows the base CoRIM schema to be customised to fit a specific Attester, though it MUST NOT alter the base CoRIM schema's syntax and semantics.

Reviewer 2: ```suggestion
A profile allows the base CoRIM schema to be customised to fit a specific Attester, though it MUST NOT alter the base CoRIM CDDL syntax and semantics.
```

> A profile allows the base CoRIM schema to be customised to fit a specific Attester, though it MUST NOT alter the base CoRIM schema's syntax and semantics.

Reviewer 3: fixing #208

How about this?

```suggestion
A profile allows the base CoRIM CDDL definition to be customized to fit a specific Attester augmenting the base CDDL data definition via the specified extension points or by constraining types defined. A profile MUST NOT change the base CoRIM CDDL definition's semantics, which including not changing or overloading names and numbers registered at the IANA registries specified by this document.
```

* A locator, which allows discovery of possibly related RIMs

* A profile identifier, which is used to interpret the information contained in the enclosed tags.
A profile allows the base CoRIM CDDL definition to be customized to fit a specific Attester by augmenting the base CDDL data definition via the specified extension points or by constraining types defined.

> A profile MUST NOT change the base CoRIM CDDL definition's semantics, which including not changing or overloading names and numbers registered at the IANA registries specified by this document.

Reviewer: ```suggestion
A profile MUST NOT change the base CoRIM CDDL definition's semantics, which includes not changing or overloading names and numbers registered at IANA registries used by this document.
```

> or more, see {{sec-corim-profile-types}},

Reviewer: ```suggestion
For more detail, see {{sec-corim-profile-types}},
```
